### PR TITLE
#96 - Add prefix during download, not use prefix when pushing

### PR DESCRIPTION
**Pull request by genryxy**
Part of #96 
Stopped using prefix in `PushChartSlice` because now urls are got by dynamically combining. For this purpose, `DownloadIndexSlice` was introduced. 
Bumped dependency of testcontainers from 1.14.3 to 1.15.1 to fix ITs failings.

### CODE REVIEW

File: src/main/java/com/artipie/helm/http/DownloadIndexSlice.java [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+            final Set<String> entrs = index.entries().keySet();
+            entrs.forEach(
+                chart -> index.byChart(chart).forEach(
+                    entr -> {
+                        final String key = "url";
+                        final List<String> urls = (List<String>) entr.get(key);
+                        entr.put(
+                            key,
+                            urls.stream()
+                                .map(uri -> String.format("%s/%s", this.base, uri))
```
Comment (g4s8) [inline src/main/java/com/artipie/helm/http/DownloadIndexSlice.java:184]: @genryxy I'd use some `URI` specific classes such as `URI` or `URIBbuilder` for creating and validating `URI` entries

**Review by g4s8 — CHANGES_REQUESTED** [review]
@genryxy please check my comment

**Comment by genryxy** [comment]
@g4s8 thanks, corrected

**Review by g4s8 — APPROVED** [review]
@genryxy thanks

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @g4s8[/z](https://www.0crat.com/u/g4s8)[/z](https://www.0crat.com/u/g4s8) didn't make CR comments

**Pull request merged**